### DISCUSSION TITLE
Recurring Payment for Nuvei using HyperSwitch  | Chargebee

### DISCUSSION THREAD

**Discussion by cb-alfredjoseph** [juspay/hyperswitch#3456] [Ideas / Feature Requests]
Hi Team,

We are trying to implement Nuvei via Hyper switch in Chargebee 

We could see, currently Hyperswitch does not support recurring payments for Nuvei.

 is there any development in Roadmap already ?

**Comment by VedantKhairnar** [comment]
hello Alfred,
Thanks for creating this discussion.
Regrettably, at present, we do not provide support for recurring payments for [Nuvei](https://hyperswitch.io/pm-list). 
.
While it is not currently included in our roadmap(Q1'24), we remain open to considering it as a priority if it emerges as a prevalent requirement among our community users. 
Your feedback is valuable, and we appreciate your understanding.

**Comment by VedantKhairnar** [comment]
Hey @cb-alfredjoseph ,
Also, if you/someone from your team would be interested in contributing to this part of Hyperswitch so as to enable the support for recurring payments for Nuvei, we would be happy to assist you whenever required for the contribution.
Thanks.

**Comment by cb-alfredjoseph** [comment]
Hi @VedantKhairnar,
Yes, We are ready to contribute for Nuvei recurring payments, can you provide us the next steps?
Do we have any docs for this ?
